### ISSUE TITLE
[Feature Request] Use "mobile" configuration for iOS Xamarin embedded webview

### ISSUE THREAD

**Issue by henrik-me** [AzureAD/microsoft-authentication-library-for-dotnet#1552]
Apple’s iPadOS (iOS 13 for iPad) switched to use macOS user-agent for all browsers by default.
This has caused some disruption in conditional access scenarios that depend on the User-Agent string. 

For UIWebView no changes are required

**To use Mobile user-agent with WKWebView when running on iPadOS see below sample code:**
```obj-c
WKWebViewConfiguration *config = [WKWebViewConfiguration new];
config.defaultWebpagePreferences.preferredContentMode = WKContentModeMobile;
WKWebView *webView = [[WKWebView alloc] initWithFrame:rootView.frame configuration:config];
```
Note: To explore the same change for ADAL

**Comment by bgavrilMS** [comment]
@henrik-me - do you have more details around this? What is the scenario that you are talking about?

**Comment by jennyf19** [comment]
released in 4.8.1
